sql/datetimetype.go

```diff
@@ -30,7 +30,7 @@ const (
 
 	// TimestampDatetimeLayout is the formatting string with the layout of the timestamp
 	// using the format of Go "time" package.
-	TimestampDatetimeLayout = "2006-01-02 15:04:05.999999"
+	TimestampDatetimeLayout = "2006-01-02 15:04:05"
```

Comment:
We need to add parameter support for `TIMESTAMP` and `DATETIME` types. In the meantime, we've chosen to use the full decimal portion until parameters are properly added.

https://dev.mysql.com/doc/refman/8.0/en/fractional-seconds.html

Rather than this change, add a `TODO` comment and file an issue.